docs_user/modules/proc_preparing-RHOSO-for-director-operator-adoption.adoc

```diff
@@ -0,0 +1,334 @@
+[id="preparing-RHOSO-for-director-operator-adoption_{context}"]
+
+= Preparing RHOSO for director Operator adoption
+
+When you deploy a {rhos_long} environment by using two {rhocp_long} master nodes, you must perform the following actions:
+
+* Create a new namespace for {rhos_acro} installation.
+* Create custom node network configuration policies.
+* Create `NetworkAttachmentDefinition` custom resources for each isolated network.
+* Install {rhos_acro} operators.
+
+.Procedure
+
+. Save the hostnames of the remaining two nodes to the `RHOSO_NODES` variable:
++
+----
+$ RHOSO_NODES=$(oc get nodes -o name | grep -v $CONTROLLER_NODE | sed 's#node/##g' | tr '\n' ' ')
+----
+. Extract nodes to be used for OSP 18 installation and label nodes:
++
+----
+$ export RHOSO18_NODE1=$(echo "${RHOSO_NODES}" | cut -d ' ' -f 1)
+$ export RHOSO18_NODE2=$(echo "${RHOSO_NODES}" | cut -d ' ' -f 2)
+$ oc label nodes "${RHOSO18_NODE1}" type=openstack
+$ oc label nodes "${RHOSO18_NODE2}" type=openstack
+----
+
+. Create a new namespace to use to install {rhos_acro}. You must install {rhos_acro} in a different namespace from OSPdO:
```

Comment:
What is OSPd0?

Reply:
director Operator yes? I had forgotten what it was. I presume it's defined elsewhere.